torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -748,7 +748,7 @@ def _recover_linear():
                 ),
                 KeywordArg("reshape_2"),
             ),
-            pass_number=1,
+            pass_number=2,
```

Comment:
Explain the change?

Reply:
Previously `mkldnn_linear` pass is 0 and `_recover_linear` is based on `mkldnn_linear` so it should be 1.
Now we set `mkldnn_linear` pass to 1 (since concat linear is 0), then the `recover_linear` pass number should be 2.

Reply:
Please add the note in the code too.

Reply:
Thanks for advice, added this note in code.